src/Model/ModelData.php

```diff
@@ -346,67 +349,15 @@ public function castingHelper(string $field, bool $useFallback = true): ?string
         // If no specific cast is declared, fall back to failover.
         $failover = $this->getFailover();
         if ($failover) {
-            $cast = $failover->castingHelper($field, $useFallback);
+            $cast = $failover->castingHelper($field);
             if ($cast) {
                 return $cast;
             }
         }
 
-        if ($useFallback) {
-            return $this->defaultCastingHelper($field);
-        }
-
         return null;
     }
 
-    /**
-     * Return the default "casting helper" for use when no explicit casting helper is defined.
-     * This helper will be a subclass of DBField. See castingHelper()
-     */
-    protected function defaultCastingHelper(string $field): string
```

Comment:
This method was also an early addition to CMS 6 (used to just get the `default_cast` config directly from `castingHelper`), so we don't need to deprecate this either.